test/open-shift/pod-kogito-gatling-benchmark.yaml

```diff
@@ -34,3 +34,18 @@ spec:
       volumeMounts:
        - name: shared-file-storage
          mountPath: /src/kogito-gatling/target
+    - name: persist-gatling-reports-to-mongodb
+        image: quay.io/almoglevi0/persist-gatling-reports-to-mongodb
```

Comment:
I think you also need to describe how this image was generated (Dockerfile and related procedures)

Reply:
thanks, 
I described the build&run procedures in the separate (persist-gatling-reports-to-mongodb) project. 
let me know if more additions need to be done :)

Reply:
Do you mean it's in a separate git repository?

Reply:
yes [https://github.com/AlmogLevii/persist-gatling-reports-to-mongodb/blob/master/README.md](https://github.com/AlmogLevii/persist-gatling-reports-to-mongodb/blob/master/README.md)

Reply:
I see
Maybe, if we want to have an automatic generation of the testing system in the automation setup of the Middleware Performance team, we'd better integrate this repo inside the other one
Let's discuss this in the next meeting
Good job for now: can we have a demo tomorrow and compare the output with the data stored in Nooba to see they are consistent?

Reply:
ok, 
I still did not run the all containers sequence as we need if its what you mean (I can to give it a try tomorrow) 
but the project that I link before works well locally.